commands.yaml

```diff
@@ -18,21 +18,21 @@
   check_command: |
     defaults read /Library/Preferences/com.apple.Bluetooth ControllerPowerState | grep 0
   fix_command: |
-    sudo defaults write /Library/Preferences/com.apple.Bluetooth ControllerPowerState -int 0 &&  sudo killall -HUP blued
+    sudo defaults write /Library/Preferences/com.apple.Bluetooth ControllerPowerState -int 0; sudo killall -HUP blued
   enabled: true
 
 
 - title: "Disable infrared receiver"
   check_command: |
     defaults read /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled | grep 0
   fix_command: |
-    defaults write /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled -int 0
+    sudo defaults write /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled -int 0
   enabled: true
 
 
 - title: "Disable AirDrop"
   check_command: |
-    defaults read com.apple.NetworkBrowser DisableAirDrop | grep 1
+    sudo defaults read com.apple.NetworkBrowser DisableAirDrop | grep 1
```

Comment:
I'm of the belief that this tool should always be run as root/sudo, but people keep requesting these sudo additions and it doesn't hurt anything.  However, I'm not testing running this as an unprivileged user, so eventually I might just make some sort of "is root" check at the top of the program and get rid of all these.